draft-ietf-quic-transport.md

```diff
@@ -3824,20 +3824,21 @@ were applied to packets that are newly acknowledged.
 ECN validation involves the following checks:
 
 * If this ACK frame newly acknowledges a packet that the endpoint sent with
-  either ECT(0) or ECT(1) codepoints set, ECN feedback MUST be present in the
-  ACK frame.  This step protects against both a network element that zeroes out
-  ECN bits and a peer that is unable to access ECN markings, since the peer
-  could respond without ECN feedback in either case.
-
-* For validation to succeed, the sum of the increases to ECT(0), ECT(1), and
-  CE counts in the ACK frame MUST be at least the number of newly acknowledged
-  packets that were sent with an ECT codepoint.  This step detects any network
-  remarking from ECT(0), ECT(1), or CE codepoints to Not-ECT.
-
-* Any increase in either ECT(0) or ECT(1) counts, plus any increase in the CE
-  count, MUST NOT exceed the number of packets sent with the corresponding ECT
-  codepoint that are newly acknowledged in this ACK frame.  This step detects
-  any erroneous network remarking from ECT(0) to ECT(1) (or vice versa).
+  either ECT(0) or ECT(1) codepoints set, ECN validation fails if ECN counts
+  are not present in the ACK frame.  This step protects against both a network
+  element that zeroes out ECN bits and a peer that is unable to access ECN
+  markings, since the peer could respond without ECN feedback in either case.
+
+* ECN validation fails if the sum of the increases to ECT(0), ECT(1), and CE
+  counts in the ACK frame are less than the number of newly acknowledged
+  packets that were sent with an ECT codepoint.  This step detects any
+  network remarking from ECT(0), ECT(1), or CE codepoints to Not-ECT.
```

Comment:
As you use the word "erroneous" below you should also use it here, however, effectively it unknown if this is intentional or not. Most cases we observes it old boxes that still set the whole byte as ToS field. I would actually also recommend to use the term "overwriting" instead of "remarking" as that probably more indicates that, not matter why this happens, it shouldn't.